### ISSUE TITLE
关于action_head中位置编码部分实际使用周期的问题

### ISSUE THREAD

**Issue by GrasRO** [X-Square-Robot/wall-x#89]
在model/action_head.py里，位置编码的部分函数中，min_period和max_period的数值并没有被forward实际的使用，
而在调用的时候，则将一个数值范围在(0,1]之间的时间传入了这个使用了泛用参数（emb = math.log(10000) / (half_dim - 1)）的位置编码，这样在理论上不是会降低对于噪声编码的精度吗？是我的理解有误，还是自变量特意的采用了这个方式？

684行：
            # 1. add noise to action_chunk
            noise = torch.randn_like(action_chunk)
            time = self.sample_time(batch_size, device, dtype)
            time_expanded = time.unsqueeze(-1).unsqueeze(-1)
            noisy_action = (1 - time_expanded) * noise + time_expanded * action_chunk
            flow = action_chunk - noise

            # 2. sinusoidal positional encoding for timesteps
            time_embed = self.time_embed(time).to(torch.float32)

class SinusoidalPosEmb(nn.Module):
    def __init__(self, dim: int, min_period: float = 4e-3, max_period: float = 4.0):
        super().__init__()
        self.dim = dim
        if dim % 2 != 0:
            raise ValueError(f"embedding_dim ({dim}) must be divisible by 2")
        self.min_period = min_period
        self.max_period = max_period

    def forward(self, x):
        device = x.device
        half_dim = self.dim // 2
        emb = math.log(10000) / (half_dim - 1)
        emb = torch.exp(
            torch.arange(half_dim, device=device, dtype=torch.float32) * -emb
        )
        emb = x[:, None] * emb[None, :]
        emb = torch.cat((emb.sin(), emb.cos()), dim=-1)
        return emb
